openhtf/plugs/frontline/bluetooth_sniffer.py

```diff
@@ -0,0 +1,277 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Classes for controlling Frontline bluetooth sniffers.
+
+This module provides control over a variety of Frontline (now owned by Teledyne
+Lecroy) bluetooth sniffers. For full details on Frontline's sniffer control
+protocol, which this module partially implements, see the reference document:
```

Comment:
doc link is good but a quick summary of the bit implemented here would be pretty handy in the docstring i think.

Reply:
Beefed up the docstring with better description of what's here plus examples.